jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -118,6 +118,18 @@
         </group>
         <action class="software.aws.toolkits.jetbrains.core.SettingsSelectorAction" id="aws.settings.selector" icon="AwsIcons.Logos.AWS"/>
         <action class="software.aws.toolkits.jetbrains.core.credentials.CreateOrUpdateCredentialProfilesAction" id="aws.settings.upsertCredentials" icon="AwsIcons.Logos.AWS"/>
+
+        <group id="aws.toolkit.serverless.template">
+            <add-to-group group-id="ProjectViewPopupMenu" anchor="before"
```

Comment:
I'm not sure what happens when GitHub plugin isnt installed since this action id wont exist

Reply:
Me either - I'll change this to use the bottom of the menu

Reply:
![image](https://user-images.githubusercontent.com/39839589/48165943-0f0bb080-e29c-11e8-95db-5edce19e06bc.png)